src/providers/ldap/ldap_child.c

```diff
@@ -408,7 +409,7 @@ static errno_t select_principal_from_keytab(TALLOC_CTX *mem_ctx,
     }
 
 done:
-    if (ret != EOK) {
+    if (ret != EOK && krb_ctx != NULL) {
         DEBUG(SSSDBG_FATAL_FAILURE, "Failed to read keytab [%s]: %s\n",
               sss_printable_keytab_name(krb_ctx, keytab_name),
```

Comment:
Would it be possible to skip keytab name only but still print error message?

Reply:
Hi,

I moved the check into `sss_printable_keytab_name()`.

bye,
Sumit